src/Matcher_lexer.mll

```diff
@@ -64,6 +64,22 @@ rule token = parse
 | "del" { BINDING (Delete) }
 | "delete" { BINDING (Delete) }
 | "bs" { BINDING (Backspace) }
+| "lt" { BINDING (LessThan) }
+| "gt" { BINDING (GreaterThan) }
+| "minus" { BINDING (Minus) }
+| "plus" { BINDING (Plus) }
+| "(" { BINDING (LeftParen) }
+| ")" { BINDING (RightParen) }
+| "|" { BINDING (Pipe) }
```

Comment:
Why do all these need to be special cased? Can you not just extend the definition of a "character"?

Reply:
Good question - I started by bringing over all the SDL Virtual Keys as new entries. I'll move these to be represented by `Character` 👍